runtime/etc/Makefile.mli-static

```diff
@@ -25,30 +25,33 @@ CHPL_MLI_INCS = -I$(CHPL_ZMQ_LOC)/include
 CHPL_LN_LIB_DIR = $(CHPL_MAKE_RUNTIME_LIB)/$(CHPL_MAKE_LAUNCHER_SUBDIR)
 UNAME = $(shell uname)
 CHPL_LIBTOOL = libtool
+CHPL_LIBTOOL_CMD =
 #
 # Assume that we are using GNU libtool by default.
 #
 CHPL_LIBTOOL_CMD = $(CHPL_LIBTOOL) --quiet --mode=link $(CC) -static
 
-.PHONY: setLibtool buildUserObj buildServer buildClient runTagsCommand
+.PHONY: setLibtoolCmd buildUserObj buildServer buildClient runTagsCommand
 
-all: setLibtool buildUserObj buildServer buildClient runTagsCommand FORCE
+all: setLibtoolCmd buildUserObj buildServer buildClient runTagsCommand FORCE
 
-setLibtool:
 #
 # On some platforms (Darwin in particular), libtool cannot be assumed to mean
 # GNU libtool and is actually a different executable that goes by the same
 # name. Darwin's libtool has similar semantics to GNU libtool, but has to be
 # invoked differently.
 #
+setLibtool:
 ifeq ($(UNAME),Darwin)
-	CHPL_LIBTOOL_CMD = libtool -static -no_warning_for_no_symbols
+	CHPL_LIBTOOL_CMD += $(CHPL_LIBTOOL) -static -no_warning_for_no_symbols
```

Comment:
I think you don't need this to be a make target, it could just be a Makefile variable that gets set based on the `ifeq` comparison

Reply:
You can have an ifeq outside of a target body?

Reply:
Yeah, that's what I did in #13122

Reply:
Well today I learned...thanks!